File: src/Ejercicio7/Ejercicio7.java
package Ejercicio7;

import java.util.Scanner;

public class Ejercicio7 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese una cadena: ");

        String cadena = sc.nextLine();
        String reverso = "";

        for (int i = cadena.length()-1; i >= 0 ;  i-- ) {
            reverso += cadena.charAt(i);
        }

        System.out.println(reverso);


    }
}

File: src/Ejercicio7/Ejercicio7.kt
package Ejercicio7

fun main(){

    print("Ingrese una cadena: ")
    val cadena = readln()
    var reverso = ""

    for (i in cadena.length-1 downTo 0 ){
        reverso += cadena[i]
    }

    println("Cadena inversa: $reverso")

}